### ISSUE TITLE
"Little Washu" by Alison (2001)

### ISSUE THREAD

**Issue by emhoracek** [emhoracek/smooch#11]
Colors are just a little bit wrong. The teal parts of the background image are transparent? or missing? and leggings are the wrong colors. This may have something to do with some other color problems in various sets, especially in background cels.

In UltraKiss: 
![screen shot 2017-08-10 at 10 13 17 am](https://user-images.githubusercontent.com/5353499/29174566-af348f62-7db4-11e7-9dc0-8c192bfebd58.png)

In Smooch:
![screen shot 2017-08-10 at 10 13 29 am](https://user-images.githubusercontent.com/5353499/29174642-e17cc6e2-7db4-11e7-8c2c-7366e7f576ff.png)

There's only a single palette for this set, so unlike "Lucca" (#10), this isn't a problem of using the wrong palette. The palette and/or cel files are getting translated wrong.

This set is in the samples directory at `/haskell/tests/samples/wasua.lzh`.

**Comment by huggablemonad** [comment]
The leggings look a little different in fkiss-0.33a:

![fkiss-washu](https://user-images.githubusercontent.com/17948023/29239914-1c43d3be-7f8c-11e7-84ed-94c14174c420.png)

**Comment by vyv03354** [comment]
fkiss does not support true translucency. It emulates transparency with tiled pattern, so it is expected.

**Comment by huggablemonad** [comment]
Oh, I didn't know that. Thanks for the info!

If I disable the "pseudo transparency mode," then it looks like Smooch's:

![fkiss-washu-no-transparency](https://user-images.githubusercontent.com/17948023/29240308-eab648f0-7f94-11e7-8611-3972dcefbab9.png)

**Comment by emhoracek** [comment]
This is really helpful, learning that it's an issue with transparency. Thanks @vyv03354 and @huggablemonad!

**Comment by emhoracek** [comment]
Wow! I just realized that I wasn't actually using a cel's alpha value at all in the JavaScript! XD This is now fixed and the colors of all the clothes look 👍 : 

<img width="545" alt="screen shot 2017-08-12 at 8 18 37 am" src="https://user-images.githubusercontent.com/5353499/29240634-1912cae2-7f37-11e7-8027-c4887c0a01d7.png">

However, the background image problem remains. The background could be that cyan color.
